### PR TITLE
add chapter 31 / 6 form

### PR DESCRIPTION
**Pull request by rianfowler**
[Design](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/8832)

Haven't noticed the light blew line that appears when using `expandWith`. Assuming that's intentional since it's animates. 

2018-02-22-14-47-localhost_3001

2018-02-22-14-45-localhost_3001 1

### CODE REVIEW

File: test/vre/chapter36/config/contactInformation.unit.spec.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import formConfig from '../../../../src/js/vre/chapter36/config/form.js';
 
 describe('VRE chapter 36 contact information', () => {
   const { schema, uiSchema } = formConfig.chapters.contactInformation.pages.contactInformation;
-  it('renders applicant address form', () => {
+  it('renders contact information form', () => {
```
Comment (rianfowler) [inline test/vre/chapter36/config/contactInformation.unit.spec.jsx]: fixing some copy pasta

File: src/js/vre/chapter31/config/form.js [hunk 2]
```diff
@@ -115,12 +124,62 @@ const formConfig = {
...
+        veteranAddress: {
+          path: 'veteran-address',
+          title: 'Address information',
+          uiSchema: {
+            veteranAddress: address.uiSchema(),
+            'view:isMoving': {
+              'ui:title': 'Are you moving within the next 30 days?',
+              'ui:widget': 'yesNo'
+            },
+            veteranNewAddress: _.merge(
```
Comment (rianfowler) [inline src/js/vre/chapter31/config/form.js]: Realized the superiority of `merge` over `...`: `merge` is recursive.

File: src/js/vre/chapter31/config/form.js [hunk 3]
```diff
@@ -159,12 +166,62 @@ const formConfig = {
...
+          uiSchema: {
+            daytimePhone: phoneUI('Daytime phone number'),
+            eveningPhone: phoneUI('Evening phone number'),
+            email: {
+              'ui:title': 'Email address'
+            },
+            'view:confirmEmail': {
+              'ui:title': 'Re-enter email address',
+              'ui:options': {
+                hideOnReview: true
```
Comment (rianfowler) [inline src/js/vre/chapter31/config/form.js]: Should I be adding this to other booleans like `view:isMoving`

Comment (jbalboni) [reply src/js/vre/chapter31/config/form.js]: I think you can leave it off of isMoving. We mainly use this for data that isn't useful to review.

**Review by jbalboni — APPROVED** [review]
LGTM, though I haven't check the build failures.

**Review by stacyhlinn — APPROVED** [review]

**Review by cehsu — APPROVED** [review]

**Pull request merged**